spec/models/rubric_criterion_spec.rb

```diff
@@ -177,65 +179,65 @@
       before(:each) do
         # we'll need a valid assignment for those cases.
         @assignment = create(:assignment)
-        row = ['criterion 5', '1.0']
-        (0..RubricCriterion::RUBRIC_LEVELS - 1).each do |i|
+        row = ['criterion 5']
+        # get number of required fields
+        required_fields = Level.validators.grep(ActiveRecord::Validations::PresenceValidator).length
```

Comment:
See my comment about what `RUBRIC_LEVELS` really refers to. Here you should be able to choose in the test case how many levels you want to create, it doesn't depend on the number of required fields for a Level. You have the right idea in a test you wrote further down.